classes/classes/Creature.as

```diff
@@ -381,7 +381,7 @@ package classes
 		//MALE STUFF
 		//public var cocks:Array;
 		//TODO: Tuck away into Male genital class?
-		public var cocks:Array;
+		public var cocks:/*Cock*/Array;
```

Comment:
I'm a bit tired right now to look for myself, but maybe you could consider vectors for stuff, like these?